lxde-base/lxdm/lxdm-0.5.3-r1.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
no revision bump is necessary here. The ebuild is unstable, and the added patch only fixes a build failure.